evap/evaluation/views.py

```diff
@@ -94,3 +101,26 @@ def faq(request):
 
 def legal_notice(request):
     return render(request, "legal_notice.html", dict())
+
+
+@require_POST
+def feedback_send(request):
+    sender_email = request.POST.get("sender_email")
```

Comment:
i addressed this in https://github.com/janno42/EvaP/commit/323f776c010b64a78487c639dba9593892fa4d87. please use the email address of the currently logged in user. unauthenticated users don't need to see the feedback button.